playbooks/ceph.yml

```diff
@@ -63,7 +63,13 @@
   become: true
   vars:
     cifmw_admin_user: ceph-admin
-    ansible_ssh_private_key_file: "{{ lookup('env', 'ANSIBLE_SSH_PRIVATE_KEY') }}"
+    _target_group: "{{ cifmw_ceph_target | default('computes') }}"
+    _target: "{{ groups[_target_group] | default([]) | first }}"
+    ansible_ssh_private_key_file: >-
+      {{
+        hostvars[_target]['ansible_ssh_private_key_file'] |
+        default(lookup('env', 'ANSIBLE_SSH_PRIVATE_KEY'))
+      }}
```

Comment:
these changes look like they will be fine but could we please test project them downstream to make sure they do not affect the unijobs?

Reply:
yes I am testing it now